ethereum/core/src/main/java/org/hyperledger/besu/ethereum/transaction/CallParameter.java

```diff
@@ -199,18 +199,61 @@ public int hashCode() {
         blobVersionedHashes);
   }
 
+  @Override
+  public String toString() {
+    return "CallParameter{"
+        + "from="
+        + from
+        + ", to="
+        + to
+        + ", gasLimit="
+        + gasLimit
+        + ", maxPriorityFeePerGas="
+        + maxPriorityFeePerGas.map(Wei::toHumanReadableString).orElse("N/A")
+        + ", maxFeePerGas="
+        + maxFeePerGas.map(Wei::toHumanReadableString).orElse("N/A")
+        + ", maxFeePerBlobGas="
+        + maxFeePerBlobGas.map(Wei::toHumanReadableString).orElse("N/A")
+        + ", gasPrice="
+        + (gasPrice != null ? gasPrice.toHumanReadableString() : "N/A")
+        + ", value="
+        + (value != null ? value.toHumanReadableString() : "N/A")
+        + ", payloadSize="
+        + (payload != null ? payload.size() : "null")
+        + ", accessListSize="
+        + accessList.map(List::size)
```

Comment:
is this auto-generated? `accessList.size()` would achieve the same result?

Reply:
no because `accessList` is `Optional`